package.json

```diff
@@ -10,7 +10,7 @@
   },
   "packageManager": "pnpm@9.4.0",
   "scripts": {
-    "dev": "nodemon --config nodemon.config.json -x 'pnpm build:packages'",
+    "dev": "nodemon --config nodemon.config.json -x \"pnpm build:packages\"",
```

Comment:
Could you please let us know if this change is aimed at resolving any specific issues, such as compatibility problems on Windows?

Reply:
Yes, I indeed encountered a problem on the Windows system. If this issue does not require modifications, I can revert the corresponding code.